Update models.py

Temporary-summer opened the pull request:
Defining the structure and fields of the Donor Registration model

Temporary-summer committed:
Update models.py

Defining the structure and fields of the Donor Registration model

Temporary-summer committed:
Update forms.py

Adding a Donor Register Form for registering a donor.

Temporary-summer committed:
Update urls.py

Updated navigation URLs for donor pages

SuyesSapkota commented:
Would you please consider about using the Programme By Contract and Literate Programming ideas in your coding?

Temporary-summer committed:
Update forms.py

Added the comments by Incorporating the principles of program by contract and Literate programming

Temporary-summer committed:
Update models.py

Adding comments by incorporating the principles of program by contract and literate programming.

SuyesSapkota reviewed (approved):
Great job on the code! It's ready for merging into the main branch.
